pkg/constants/constants.go

```diff
@@ -120,6 +120,8 @@ const (
 	PipelineRunPollingInterval = 10 * time.Second
 
 	JsonStageUsersPath = "users.json"
+	// Github App name is RHTAP-Qe-App
+	DEFAULT_PAC_GITHUB_APP_ID = "310332"
```

Comment:
```suggestion
	DefaultPaCGitHubAppID = "310332"
```
no need to name this with capitals as it is not used as an env var

Reply:
fixed